src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,45 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin onTablet {
+  @media (min-width: 744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
```

Comment:
Consider defining a variable for the desktop breakpoint (e.g., $desktop-breakpoint: 1260px) and use it in the onDesktop mixin.